pkg/cover/cover.go

```diff
@@ -283,14 +283,21 @@ func CovList(f io.Reader) (g *CoverageList, err error) {
 	return
 }
 
-// NewCoverageList constructs new (file) group Coverage
-func NewCoverageList(name string, concernedFiles map[string]bool, covThresholdInt int) *CoverageList {
-	return &CoverageList{
-		Coverage:        newCoverage(name),
-		Groups:          []Coverage{},
-		ConcernedFiles:  concernedFiles,
-		CovThresholdInt: covThresholdInt,
+// covert profile file to CoverageList struct
+func ReadFileToCoverList(path string) (g *CoverageList, err error) {
+	f, err := ioutil.ReadFile(path)
+	if err != nil {
+		logrus.Errorf("Open file %s failed!", path)
+		return nil, err
 	}
+	g, err = CovList(bytes.NewReader(f))
+	return
+}
+
+// NewCoverageList return empty CoverageList
+func NewCoverageList() *CoverageList {
```

Comment:
it's better to be consistent for the type of struct `CoverageList `, including it's methods。like:
```
func (g *CoverageList) size() int {}
func (g CoverageList) lastElement() *Coverage {}
```
In my opinion, suggest to use `CoverageList` instead of `*CoverageList`

Reply:
CoverageList is slice struct, will use CoverageList instead * CoverageList except append operation.